scopes/dependencies/dependency-resolver/dependency-installer.ts

```diff
@@ -314,6 +316,7 @@ export class DependencyInstaller {
       manifests[rootDir] = workspaceManifest.toJson({
         copyPeerToRuntime: copyPeerToRuntimeOnRoot,
         installPeersFromEnvs,
+        ignoredPeers: allPkgNames,
```

Comment:
I'm wondering about this ignore, why do you need to ignore these?
won't this cause us to use the version from the component even if the env declares a specific version?

Reply:
If you don't ignore it, then it will be added to the "dependencies" field, which will effectively make them non-peer dependencies.

> won't this cause us to use the version from the component even if the env declares a specific version?

It works correctly with package peer dependencies at the moment and will work the same way with component peer dependencies.

Reply:
do we ignore the package peer deps the same way now?

Reply:
I have reverted it. The tests passed. So, I guess it is not needed.